packages/core/src/config/config-resolvers.ts

```diff
@@ -342,7 +346,7 @@ async function loadExtendStyleguideConfig(
 
     return rawConfig.styleguide;
   } catch (error) {
-    throw new Error(`Failed to load "${filePath}": ${error.message}`);
+    throw new Error(`Failed to load "${filePath}": Please provide a valid path`);
```

Comment:
Why don't we want to keep the generic `error.message` here? In future there might be different errors inside the function, not necessarily related to  a wrong path.